pkg/innerring/indexer.go

```diff
@@ -52,6 +53,11 @@ func (s *innerRingIndexer) update() (err error) {
 		return err
 	}
 
+	s.alphabetIndex, err = invoke.AlphabetIndex(s.cli, s.key)
```

Comment:
I am not sure what desired behavior should be. If inner ring or alphabet keys are unavailable -- it is very-very bad state of the network. It might be not safe to continue work in such network. Thus I decided to proceed only if all values are fetched correctly.